### PR TITLE
[REVIEW] Add CUDA label to conda install

### PR DESCRIPTION
**Pull request by dillon-cullinan**

### CODE REVIEW

**Comment by dillon-cullinan** [comment]
rerun tests

**Review by raydouglass — APPROVED** [review]

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ function logger() {
 # Set path and build parallel level
 export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
 export PARALLEL_LEVEL=4
+export CUDA_REL=${CUDA_VERSION%.*}
```
Comment (raydouglass) [inline ci/gpu/build.sh:16]: Why `CUDA_VERSION`? gpuCI uses `CUDA`.

**Review by raydouglass — CHANGES_REQUESTED** [review]

**Comment by dillon-cullinan** [comment]
rerun tests

File: ci/gpu/build.sh (hunk 1 above)
Comment (dillon-cullinan) [reply ci/gpu/build.sh:16]: gpuCI uses `CUDA_VERSION` from what I see when the environment is printed. Just tried using `CUDA` and its undefined.

**Review by raydouglass — APPROVED** [review]

**Pull request merged**